### PR TITLE
Issue #1690

### LINKED ISSUE #1690: Python Implementation of Circular Queue
I would like to contribute this as a part of GSSoC 2020. Please assign me the same.

### PR DESCRIPTION
**Pull request by Garimagupta85**
Fixes #1690 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added python Implementation of circular queue

#### Languages Used:
- python3

#### Files Added:
- Circular_Queue.py

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by Garimagupta85** [comment]
@asha15 Please review

File: Circular_Queue/Circular_Queue.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
+class CircularQueue(object):
+	'''Simple Circular Queue'''
```
Comment (jainaman224) [inline Circular_Queue/Circular_Queue.py]: Use 4 space indentation.

File: Circular_Queue/Circular_Queue.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+		'''
+		if ((self._rear + 1) % self.Max == self._front):
+			return False 
+
+		elif (self._front == -1):
+			self._front = 0
+			self._rear = 0
+			self.queue[self._rear] = key
+
+
```
Comment (jainaman224) [inline Circular_Queue/Circular_Queue.py]: Remove extra new lines.

File: Circular_Queue/Circular_Queue.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+		else:
+			self._front = (self._front + 1) % self.Max
+			self._size -= 1
+
+		if (self._front == -1):
+			self._front = -1
+			self._rear = -1
+
+		return True
+	def Front(self):
```
Comment (jainaman224) [inline Circular_Queue/Circular_Queue.py]: Add a new line above function definition.

File: Circular_Queue/Circular_Queue.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+			for i in range(self._front, self.Max):
+				print(self.queue[i], end = " ")
+			for i in range(0, self.rear + 1):
+				print(self.queue[i], end = " ")
+			print ()
+		if ((self._rear + 1) % self.Max == self._front):
+			print("Queue is Full") 
+
+# Sample Input / Output
+obj = CircularQueue(6)
```
Comment (jainaman224) [inline Circular_Queue/Circular_Queue.py]: Take input from user.

File: Circular_Queue/Circular_Queue.py [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+p1 = obj.Enqueue(5)
+p6 = obj.Enqueue(7)
+p7 = obj.Enqueue(8)
+p8 = obj.Enqueue(9)
+p2 = obj.Dequeue()
+p3 = obj.Dequeue()
+p8 = obj.Enqueue(10)
+p5 = obj.PrintQueue()
```
Comment (jainaman224) [inline Circular_Queue/Circular_Queue.py]: Add sample input and output.

**Comment by jainaman224** [comment]
@Garimagupta85 Are you working on it?

**Comment by somya-kapoor** [comment]
@Garimagupta85 Are you working on it?

**Comment by somya-kapoor** [comment]
@Garimagupta85 Due to no activity, I am closing this issue

**Pull request closed without merging**